src/test/scala/spray/json/PrettyPrinterSpec.scala

```diff
@@ -50,17 +55,17 @@ class PrettyPrinterSpec extends Specification {
           |  "number": -0.000012323424,
           |  "simpleKey": "some value",
           |  "sub object": {
-          |    "sub key": 26.5,
           |    "a": "b",
           |    "array": [1, 2, {
-          |      "yes": 1,
-          |      "no": 0
-          |    }, ["a", "b", null], false]
+          |      "no": 0,
```

Comment:
New rendering output is now ordered by the key Strings. However, that's not guaranteed anywhere so I fixed the test not to assume anything.